RealTimeFaceMaskDetector/SQLServerDLL/ConfigReader.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+#ifdef SQLSERVER_EXPORTS
+#define INIPARSER_API __declspec(dllexport)
+#else
+#define INIPARSER_API __declspec(dllimport)
+#endif
+
+struct Params
+{
+	std::string param;
+	std::string value;
+};
+
+struct Section
+{
+	std::string name;
+	std::vector<Params> params;
+};
+
+class INIPARSER_API ConfigReader abstract
```

Comment:
There is no point in writing the 'abstract' key(Unless for readability). You have a pure virtual function